File: Tiles CLIENTS/cpp/examples/TileActiveListener.cpp
#include "TileActiveListener.h"
#include <iostream>
TileActiveListener::TileActiveListener()
{
	setId(100);
}

void TileActiveListener::listener(const std::string& topic,const std::string& msg)
{
	std::cout << topic << " " << msg << std::endl;
}

File: Tiles CLIENTS/python/examples/TileActiveListener.py
import tileclient

class TileActiveListener(tileclient.TileListener):
    def __init__(self):
        tileclient.TileListener.__init__(self,100)

    def listen(self,topic,data):
        print("ACTIVE LISTENER"+data)

        if(data=="true"):
            print("publishing..")
            self.publish("foo","bar")#publish to topic
